src/System.Text.Json/tests/Serialization/CustomConverterTests.DictionaryConverterForIDictionary.cs

```diff
@@ -9,17 +9,25 @@ namespace System.Text.Json.Serialization.Tests
 {
     public static partial class CustomConverterTests
     {
-        // Demonstrates custom Dictionary<string, long>; Adds offset to each integer or long to verify converter ran.
-        internal class DictionaryConverter : JsonConverter<Dictionary<string, long>>
```

Comment:
Are we losing some test case coverage by not having a concrete `Dictionary<TKey, TValue>` based converter, or do you feel confident the scenario is covered by the other tests/changes here?

Reply:
This example is for a simple Dictionary converter for specific types. The other samples provide more complicated scenarios that use the factory pattern and can support any TKey and\or TValue.

Reply:
This was an existing test - just renamed. The new samples provide for any TKey and\or any TValue.